Userland/Libraries/LibWeb/Fetch/BodyInit.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2022, Linus Groh <linusg@serenityos.org>
+ * Copyright (c) 2022, Kenneth Myhra <kennethmyhra@serenityos.org>
```

Comment:
I assume this is intentional? Just asking since there is no co-author on the commit

Reply:
`extract_body()` was written by Kenneth originally :)